app/renderer/components/Inspector/SessionInfo.js

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from 'react';
+import { Table, Card } from 'antd';
+import SessionCodeBox from './SessionCodeBox';
+import { withTranslation } from '../../util';
+
+const sessionObj = {
+  session_id: 'Session ID', session_url: 'Session URL',
+  server_details: 'Server Details', session_length: 'Session Length',
+  session_caps: 'Session Capabilities', server_response: 'Server Details from Server',
+  active_appId: 'Currently Active App ID'
+};
+
+const distanceY = 105;
+const distanceX = 100;
+const columnWidth = 200;
+
+class SessionInfo extends Component {
+
+  componentDidMount () {
+    const {driver, getActiveAppId, getServerStatus} = this.props;
+    const {isIOS, isAndroid} = driver.client;
+
+    getActiveAppId(isIOS, isAndroid);
+    getServerStatus();
+  }
+
+  generateSessionTime () {
+    const { sessionStartTime } = this.props;
+    const currentTime = Date.now();
+    const time = currentTime - sessionStartTime;
+
+    const hourDiff = time / 3600000;
+    const hours = Math.floor(hourDiff);
+
+    const minDiff = (hourDiff - hours) * 60;
+    const minutes = Math.floor(minDiff);
+
+    const secDiff = (minDiff - minutes) * 60;
+    const seconds = Math.floor(secDiff);
+
+    return `${minutes}:${(seconds < 10) ? '0' : ''}${seconds}`;
+  }
+
+  createTable (tableValues, keyName, outerTable) {
```

Comment:
`create` sounds like a verb which is producing something that has its own existence. this function is more returning a table. i would call it like `table` or `getTable`